### ISSUE TITLE
Doc should be updated

### ISSUE THREAD

**Issue by pentiumao** [robolectric/robolectric#3922]
### Description
The doc content  [Hermetic Builds with Robolectric](http://robolectric.org/blog/2017/03/01/hermetic-builds/) should be updated.

The ` Gradle build script` is not compatible with the latest version 3.8.

### Steps to Reproduce

Do as steps described not working.

### Robolectric & Android Version

Robolectric Version : 3.8
Android Version : 27

### Link to a public git repo demonstrating the problem:

old version [Gradle build script](https://gist.github.com/xian/05c4f27da6d4156b9827842217c2cd5c)
update version [Gradle build script](https://gist.github.com/pentiumao/121056319e2060c087f6465aefe82bb6)

**Comment by jongerrish** [comment]
Mind sending a pull request?

**Comment by pentiumao** [comment]
@jongerrish No,  I opened this issue because I can't find the source of the doc website.
